examples/client/client.c

```diff
@@ -1166,7 +1166,12 @@ static const char* client_usage_msg[][70] = {
         "-e          List Every cipher suite available, \n",            /* 19 */
         "-g          Send server HTTP GET\n",                           /* 20 */
         "-u          Use UDP DTLS,"
+#ifndef WOLFSSL_DTLS13
```

Comment:
Can you fix the idents so they match? Does the SCTP DTLS section below need updated too (guessing no)?